tasks/configure.yml

```diff
@@ -70,7 +70,7 @@
       dest: "/home/{{ gocd_server.user }}/bin/aws_s3_backup.sh"
     - src: aws_s3_restore.sh.j2
       dest: "/home/{{ gocd_server.user }}/bin/aws_s3_restore.sh"
-  when: gocd_server.aws.backup_bucket
+  when: "{{ gocd_server.aws.backup_bucket }}"
```

Comment:
Now, I have changed this back and two other places to get Ansible 1.9.4 and 2.1.1 working concurrently.